src/NuGet.Core/NuGet.ProjectManagement/INuGetProjectContext.cs

```diff
@@ -59,4 +61,20 @@ public enum MessageLevel
         /// </summary>
         Error
     }
+
+    /// <summary>
+    /// Enum for the type of NuGetProjectAction
+    /// </summary>
+    public enum NuGetProjectActionType
+    {
+        /// <summary>
+        /// Install
+        /// </summary>
+        Install,
```

Comment:
What do you set when the action is Update (which will uninstall and then install)?